assets/js/modules/tagmanager/components/setup/SetupForm.js

```diff
@@ -176,6 +199,16 @@ export default function SetupForm( { finishSetup } ) {
 				<WebContainerSelect />
 
 				<AMPContainerSelect />
+
+				{ isResolvingWebGetLiveContainerVersion ||
+				isResolvingAMPGetLiveContainerVersion ? (
+					<div className="googlesitekit-margin-left-1rem googlesitekit-align-self-center">
+						<small>
+							{ __( 'Checking tags…', 'google-site-kit' ) }
+						</small>
+						<ProgressBar small compress />
+					</div>
+				) : null }
```

Comment:
Rather than duplicating the logic in both places, let's extract a new component that includes the logic within it similar to others like `SetupErrorNotice`. How about `<TagCheckProgress />`. This could go in to `components/common` as it would be shared between setup and settings 👍 

As mentioned above, the classes shouldn't be necessary but if they are for some reason I'm not seeing, let's pass those as a prop here as they're contextual to this placement. Otherwise, a relevant `googlesitekit-tag-check-progress` class on the wrapping div would be fine.